tests/b2share_unit_tests/files/test_verify_checksum.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of EUDAT B2Share.
+# Copyright (C) 2017 CERN.
+#
+# B2Share is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# B2Share is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with B2Share; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Test Invenio integration with submission tests."""
+from flask import current_app
+from mock import patch
+from six import BytesIO
+from six.moves.urllib.request import urlopen
+from invenio_db import db
+from invenio_files_rest.models import Bucket, ObjectVersion
+from invenio_files_rest.storage.base import FileStorage
+from invenio_files_rest.tasks import verify_checksum
+
+from b2share_unit_tests.helpers import create_deposit, create_user, \
+    generate_record_data, url_for_file
+from b2share.modules.files.tasks import schedule_all_files_for_checksum, \
+    schedule_failed_checksum_files
+
+
+def test_verify_checksum(app, tmp_location):
+    """Test that verify_checksum sets last_check=False
```

Comment:
This docstring could be more explicit.
What about:
"""Test that the celery task verify_checksum detects wrong checksums."""